.github/actions/git-diff-on-components/src/index.js

```diff
@@ -336,20 +344,35 @@ async function run() {
     componentsDiffContents = await checkVersionModification(componentsPendingForGitDiff, componentsThatDidNotModifyVersion);
   }
 
-  const totalErrors = componentsThatDidNotModifyVersion.length + componentsDiffContents.length;
+  const totalErrors = componentsThatDidNotModifyVersion.length;
   let counter = 1;
 
+  await execCmd("git", ["config", "--global", "user.email", "lucascarezia@gmail.com"]);
+  await execCmd("git", ["config", "--global", "user.name", "Lucas Caresia"]);
+
   componentsThatDidNotModifyVersion.forEach((filePath) => {
     console.log(`${counter++}) You need to change the version of ${filePath}.`);
   });
 
-  componentsDiffContents.forEach(({ dependencyFilePath, componentFilePath }) => {
-    console.log(`${counter++}) You need to change the version of ${getComponentFilePath(componentFilePath)} since dependency file ${getComponentFilePath(dependencyFilePath)} was modified.`);
-  });
+  if (componentsDiffContents.length) {
+    for ({ dependencyFilePath, componentFilePath } of componentsDiffContents) {
+      const content = await readFile(componentFilePath, "utf-8")
+      const currentVersion = getVersion(content)
+      const increasedVersion = increaseVersion(currentVersion)
+
+      await execCmd("sed", ["-i", `0,/${currentVersion}/{s/${currentVersion}/${increasedVersion}/}`, getComponentFilePath(componentFilePath)]);
```

Comment:
For QA: We need to make sure this command doesn't change any other line besides the component version